data/config.ini

```diff
@@ -44,13 +45,17 @@ requests-per-sec=100 ;; maximal amount of requests per second in the window, 0 t
 ban-ip-sec=300 ;; time duration in seconds to ban particular IP, 0 to ban forever
 
 [upstream]
+default=127.0.0.1:28681
 blah=https://127.0.0.1:8080
 walletnode=http://127.0.0.1:28694
```

Comment:
format is very confusing and not self-explanatory, especially "keepAlive" - why just don't have it as last optional parameter? e.g.:
address[, timeout[, max_connections, [keep_alive]]]

Reply:
ok, suggesting following format:
```
<name>=<uri>[,<timeout-in-seconds>[,<max-active-connections-per-upstream>],max-active-connections-per-address,]]keep-alive]]]
```
because connections limit and keep alive are unrelated entities and right now `keepAlive` flag implicitly switches limit's scope (upstream limit vs address (aka unique host:port record))

Reply:
Comma-separated values with variable number of arguments are horrible and non-descriptive.  How about:

```
<name>=<uri>
<name>-timeout=seconds
<name>-max-conns-upstream=count
<name>-max-conns-address=count
<name>-keep-alive=seconds
```